java/test/jmri/jmrit/display/layoutEditor/PackageTest.java

```diff
@@ -24,7 +24,6 @@
         LayoutEditorActionTest.class,
         LayoutEditorAuxToolsTest.class,
         LayoutEditorFindItemsTest.class,
-        LayoutEditorLoadAndStoreTest.class,
```

Comment:
Please leave the load and store test itself in place.  We should have this test here.

We may need to adjust the data files to make it work correctly.